vars/spectrum-dark.css

```diff
@@ -1,3 +1,15 @@
+/*
```

Comment:
Everything in `vars/` is technically machine-generated (though that's not obvious here), and they'll be overwritten each time we update these vars...